index.js

```diff
@@ -80,27 +81,28 @@ module.exports = class Combobo {
 
   initEvents() {
     Emitter(this);
+    if(!this.optionsWithKeyEventHandlers.has(this.input.id)){
```

Comment:
relying on the input having an id is flawed, you can not safely assume that the input will always have an id